studio/images.d.ts

```diff
@@ -1,4 +1,4 @@
-declare module "*.webp" {
+declare module "*." {
```

Comment:
Alt sammen .* skal være lov? Da burde den ikke hete images.d.ts. Deklarer flere moduler i denne filen for andre bilder?

Reply:
Usikker på hva som har skjedd her, gikk litt fort i svingene tror jeg! Hadde ikke tenkt til å endre på denne